src/dev/build/lib/build.test.ts

```diff
@@ -35,8 +35,11 @@ expect.addSnapshotSerializer(createAbsolutePathSerializer());
 
 const config = new Config(
   true,
+  false,
+  false,
+  false,
   {
-    version: '8.0.0',
+    version: '1.0.0',
```

Comment:
How has this not caused us problems yet?

Reply:
I think it is okay to change to 1.0.0 to match opensearch dashboards. Since this is a unit test and only fetch env version. In real application, it will fetch whatever version in env. So shouldn't cause any BWC issue. Do we have any other concerns here?

Reply:
Oh yeah that makes sense.